lua/telekasten.lua

```diff
@@ -203,6 +206,33 @@ local function escape(s)
     return string.gsub(s, "[%%%]%^%-$().[*+?]", "%%%1")
 end
 
+local function recursive_substitution(dir, old, new)
+    if not global_dir_check() then
+        return
+    end
+
+    if vim.fn.executable("find") == 0 then
+        vim.api.nvim_err_write("Find not installed!\n")
+        return
+    end
+    if vim.fn.executable("sed") == 0 then
+        vim.api.nvim_err_write("Sed not installed!\n")
+        return
+    end
+
+    os.execute(
+        "find "
+            .. dir
+            .. " -type f -name '*"
+            .. M.Cfg.extension
+            .. "' -exec sed -i 's|"
+            .. old
```

Comment:
One more thing: since you're passing old and new to sed, both terms will need to be escaped, so they can't interfere with the regex. I guess there is code in Telescope for that already - search for `escape_chars` in `files.lua`.

Reply:
[files.lua:19](https://github.com/nvim-telescope/telescope.nvim/blob/0011b1148d3975600f5a9f0be8058cdaac4e30d9/lua/telescope/builtin/files.lua#L19)

Reply:
Perfect, thank you